src/Client/IO.hs

```diff
@@ -347,15 +344,15 @@ getFieldFromTzbtcUStore
   :: forall name t m. (HasTezosRpc m, HasStoreTemplateField t name)
   => m (Maybe t)
 getFieldFromTzbtcUStore =
-  getFromTzbtcUStore $ fieldNameToMText @name
+  getFromTzbtcUStore $ mkFieldMarkerUKeyL @UMarkerPlainField (fromLabel @name)
```

Comment:
why not `symbolToMText @name`? :thinking:

Reply:
`symbolToMText @name` should work only for plain fields, and e.g. for stored entrypoints, the representation will be different, I wanted to highlight that. Though it now looks a bit verbose :confused: